### ISSUE TITLE
Crash on start

### ISSUE THREAD

**Issue by chili7** [AnemoneTeam/Anemone#74]
After applying several theme elements, Anemone will no longer load. Screen opens black and then app crashes after about 10 seconds. iPad mini on ios 8.1. After reinstalling and reapplying theme elements, the same thing happened after a different theme was applied.

**Comment by cj81499** [comment]
Try safe mode?

**Comment by chili7** [comment]
Hi cj and thanks. My understanding is that safe mode lets me get out of a boot loop.  I don't have that, the app just crashes on start. If I'm missing something please let me know. I just removed Anemone and all theme elements, did a hard reset, ran iCleaner, then rebooted. Then reinstalled just Anemone to start over, then resprung, and it still crashes on start. Black screen for 10 or so seconds and then closes.

**Comment by cj81499** [comment]
Safe mode disables tweaks. This will determine if the issue is with Anemone, or a conflict with another tweak.

**Comment by chili7** [comment]
I can try that, but as I stated, Anemone installed and ran fine with the few tweaks I have. I was also able to apply the new Buufjuiced theme elements that apply to iPad without any issues and they applied correctly. However, Anemone wouldn't start after that. I also have several Anemone crash logs that appear to indicate an issue. The headers are like this:

{"os_version":"iPhone OS 8.1 (12B410)","bug_type":"188"}
Incident Identifier: 636A68BF-B933-44AF-A279-BA82E293AD53
CrashReporter Key:   ed68098f44bd0273b3dcfa872bb52619f71e5fc9
Hardware Model:      iPad2,5
OS Version:          iPhone OS 8.1 (12B410)
Kernel version:      Darwin Kernel Version 14.0.0: Tue Oct  7 00:04:32 PDT 2014; root:xnu-2783.3.13~4/RELEASE_ARM_S5L8942X
Date:                2015-12-22 21:27:28 -0700
Exception Code:      0x8badf00d
Reason:              com.anemonetheming.anemone failed to scene-create in time
Elapsed total CPU time (seconds): 21.690 (user 21.690, system 0.000), 53% CPU 
Elapsed application CPU time (seconds): 18.733, 46% CPU

I'll try safe mode and see if it makes a difference tonight.

**Comment by chili7** [comment]
In Safe Mode, with no visible tweaks in Settings, same result:

os_version":"iPhone OS 8.1 (12B410)","bug_type":"188"}
Incident Identifier: B031A683-5B8D-4A08-8061-535DDF87ABB7
CrashReporter Key:   ed68098f44bd0273b3dcfa872bb52619f71e5fc9
Hardware Model:      iPad2,5
OS Version:          iPhone OS 8.1 (12B410)
Kernel version:      Darwin Kernel Version 14.0.0: Tue Oct  7 00:04:32 PDT 2014; root:xnu-2783.3.13~4/RELEASE_ARM_S5L8942X
Date:                2015-12-23 18:44:34 -0700
Exception Code:      0x8badf00d
Reason:              com.anemonetheming.anemone failed to scene-create in time
Elapsed total CPU time (seconds): 24.990 (user 24.990, system 0.000), 57% CPU 
Elapsed application CPU time (seconds): 19.890, 45% CPU

Thermal data unavailable

Frontmost process PID:    268
Frontmost process PID:    48
Jetsam Level:              0
Free Pages:              806
Active Pages:          14500
Inactive Pages:         6905
Purgeable Pages:         876
Wired Pages:           23084
Speculative Pages:       640
Throttled Pages:       82369
File-backed Pages:     21533
Compressions:              0
Decompressions:            0
Compressor Size:           0
Busy Buffer Count:         0
Pages Wanted:            517
Pages Reclaimed:        3716

**Comment by cj81499** [comment]
Dang. No idea then. @coolstar can you help?

**Comment by coolstar** [comment]
@chili7 can you try resetting Anemone by deleting com.anemoneteam.anemone.plist and com.anemoneteam.anemoneordering.plist in /var/mobile/Library/Preferences?

**Comment by chili7** [comment]
Well that worked, sorta. I deleted the two files and Anemone came up after about 10 seconds and everything was disabled. I enabled Anemone, ClassicDock, FlipSwitch and the only theme I have currently installed, Votive, and everything applied correctly after reboot. However, trying to start Anemone again resulted in the same error. I'm starting to think the iPad mini is just too dog slow to run it. :-( Let me know if I can provide any additional info.

**Comment by cj81499** [comment]
If I'm not mistaken,the iPad Mini is more powerful than an iPod 5, which I (and I think coolstar too) use Anemone on.

**Comment by chili7** [comment]
Now that's interesting. I'm not sure what the issue is then. They appear to have identical system hardware across the board:

```
                    iPad Mini                                                iPod 5 Touch
```

System chip     Apple A5                                                    Apple A5  
Processor       Dual-core, 1000 MHz, ARM Cortex-A9  Dual-core, 1000 MHz, ARM Cortex-A9  
Graphics processor  PowerVR SGX543MP2                   PowerVR SGX543MP2  
System memory     512 MB RAM                                    512 MB RAM  
Built-in storage    64 GB                                           64 GB

**Comment by coolstar** [comment]
@cj81499 @chili7 I've tested Anemone on both iPod touch 5 (iOS 8.4) and iPad mini 1 (iOS 7.1.2) and the app launches fine on both.

**Comment by chili7** [comment]
I can reproduce the error above at will on 8.1, but there is a workaround so it's all good. It just takes a few minutes to delete the plist files and reselect the options. It also works perfectly on my iPhone 6 on 9.02 and it has even more tweaks installed

**Comment by coolstar** [comment]
0x8badf00d means that iOS thinks the initialization is taking too long and is killing the Anemone app. Something is making the app launch longer than normal for you, interesting...

**Comment by coolstar** [comment]
Is this issue still happening on the latest version from the Anemone nightly repo? (https://anemonetheming.com/nightly)

**Comment by chili7** [comment]
Yes, it still does. I added the source and updated. The crash file is below. But since I last posted I've seen a few other apps die at start up on this iPad for some reason. I should probably wipe and reimage at some point when I can take the time.  

{"os_version":"iPhone OS 8.1 (12B410)","bug_type":"188"}
Incident Identifier: 74577713-FB02-40FB-A2AC-38BC294C2310
CrashReporter Key:   ed68098f44bd0273b3dcfa872bb52619f71e5fc9
Hardware Model:      iPad2,5
OS Version:          iPhone OS 8.1 (12B410)
Kernel version:      Darwin Kernel Version 14.0.0: Tue Oct  7 00:04:32 PDT 2014; root:xnu-2783.3.13~4/RELEASE_ARM_S5L8942X
Date:                2016-05-04 20:57:43 -0600
Exception Code:      0x8badf00d
Reason:              com.anemonetheming.anemone failed to scene-create in time
Elapsed total CPU time (seconds): 26.260 (user 26.260, system 0.000), 63% CPU 
Elapsed application CPU time (seconds): 18.010, 43% CPU

**Comment by coolstar** [comment]
Does this issue still happen with 1.1-400?
